README.md

```diff
@@ -248,12 +248,12 @@ To install in development mode, use the following:
 ```bash
 $ git clone git+https://github.com/cthoyt/curies.git
 $ cd curies
-$ pip install -e .
+$ pip install -e ".[tests,fastapi,flask,pandas,rdflib,bioregistry]"
 ```
 
 ### 🥼 Testing
 
-After cloning the repository and installing `tox` with `pip install tox`, the unit tests in the `tests/` folder can be
```

Comment:
I disagree with this - tox takes care of installing the testing dependencies, and I don't want to support doing it any other way.

Reply:
I'm going to revert all of the README changes since it's distracting for this PR